### ISSUE TITLE
Enhancement idea : Mix with typegoose

### ISSUE THREAD

**Issue by xrobert35** [olivierlsc/swagger-express-ts#2]
Hello :),

I'm currently working with typegoose on a new project I started to write something to dynamically generate swagger model from Typegoose schema.

https://github.com/szokodiakos/typegoose

Here's a start idea :
```
getSwaggerProperties() {
    // Get the schema
    const userSchema = new UserSchema().getModelForClass(UserSchema);
    const user = {
      User: {
        properties: {}
      }
    };
   // Iterate throught schema properties to get swagger model
    Object.keys(userSchema.schema.obj).forEach((key) => {
      user.User.properties[key] = {
        type: userSchema.schema.obj[key].type.name.toLowerCase(),
        required: userSchema.schema.obj[key].required
      };
    });
    return user;
  }
```

What do you think about it ? 

Thanks

**Comment by olivierlsc** [comment]
Hi @xrobert35 ,
I propose to create 2 decorators :
1. @ApiModel({
name: "User"
...
})
2. @ApiProperty({
required: true
...
})
What do you think about it ?
Thanks.

**Comment by xrobert35** [comment]
Hello,

It would be great, exactly how it should be :)

Thanks

**Comment by xrobert35** [comment]
see pull request #6

**Comment by olivierlsc** [comment]
Your pull request has been merged. Thank you for your help. Are you interested in being part of the project collaborators?

**Comment by olivierlsc** [comment]
Available in version v1.0.0-rc.1
